apps/package.json

```diff
@@ -279,54 +206,128 @@
+    "lazysizes": "^4.0.0-rc1",
+    "loadable-components": "2.2.3",
+    "lodash": "^4.17.21",
     "memoize-one": "^5.1.1",
+    "messageformat": "2.3.0",
     "ml-knn": "^3.0.0",
+    "moment": "^2.29.4",
+    "newrelic": "^1.27.2",
     "object-fit-images": "^3.2.3",
     "pa11y-ci": "^2.3.0",
+    "path-browserify": "^1.0.1",
+    "pepjs": "^0.4.3",
+    "pixelmatch": "^5.2.0",
     "playground-io": "code-dot-org/playground-io#v0.6.0-cdo.2",
     "prop-types": "^15.6.2",
+    "pusher-js": "4.1.0",
     "pyodide": "^0.24.1",
     "qrcode.react": "^0.8.0",
+    "qtip2": "2.2.0",
     "query-string": "4.1.0",
+    "radium": "^0.25.2",
+    "react": "~16.14.0",
     "react-ace": "^9.2.1",
+    "react-addons-test-utils": "~15.4.0",
     "react-beautiful-dnd": "^13.1.0",
+    "react-bootstrap": "^0.33.1",
     "react-bootstrap-2": "npm:react-bootstrap@^2.7.4",
     "react-chartjs-2": "^5.2.0",
+    "react-color": "^2.17.3",
     "react-csv": "^2.0.3",
+    "react-datepicker": "1.6.0",
     "react-debounce-input": "^3.2.2",
+    "react-dom": "~16.14.0",
     "react-dom-confetti": "^0.2.0",
+    "react-google-charts": "2",
     "react-hot-loader": "^1.3.1",
+    "react-idle-timer": "^4.2.7",
+    "react-inspector": "2.3.1",
+    "react-lazy-load": "~3.0.13",
     "react-loading-skeleton": "^3.1.0",
+    "react-motion": "^0.5.2",
+    "react-onclickoutside": "~5.11.1",
+    "react-paginate": "^6.3.0",
+    "react-pointable": "^1.1.1",
     "react-portal": "^4.2.1",
+    "react-redux": "~8.0.5",
+    "react-router": "3.2.6",
     "react-router-dom": "^4.3.1",
+    "react-select": "^1.2.1",
+    "react-sticky": "^6.0.3",
+    "react-tether": "^1.0.4",
+    "react-tooltip": "^3.2.7",
     "react-transition-group": "2.9.0",
     "react-typist": "^2.0.5",
+    "react-virtualized": "^9.18.5",
+    "react-virtualized-select": "^3.0.1",
+    "react-with-context": "^2.0.0",
     "reactabular-sticky": "^8.14.0",
+    "reactabular-table": "^8.14.0",
     "reactabular-virtualized": "^8.18.0",
+    "recorder-js": "^1.0.7",
+    "redux": "^4.2.1",
+    "redux-logger": "^2.6.1",
+    "redux-thunk": "^2.0.1",
+    "rehype-raw": "^4.0.0",
+    "rehype-react": "^3.1.0",
+    "rehype-sanitize": "^2.0.2",
     "rehype-stringify": "^7.0.0",
+    "remark-rehype": "^4.0.0",
+    "rgbcolor": "0.0.4",
     "rosie": "^2.0.1",
+    "sanitize-html": "^1.11.3",
+    "save-csv": "^4.0.6",
+    "seedrandom": "2.4.2",
     "selectize": "^0.12.4",
     "slate": "^0.81.0",
     "slate-react": "^0.81.0",
-    "source-map-loader": "^4.0.1",
+    "sortabular": "^1.6.0",
+    "sprintf-js": "^1.0.3",
     "stream-browserify": "^3.0.0",
     "survey-react": "^1.9.28",
+    "swiper": "^11.0.5",
     "timers-browserify": "^2.0.12",
-    "ts-loader": "^9.4.2",
-    "typescript": "^5.3.3",
     "usb": "^2.8.1",
+    "video.js": "7.6.6",
+    "vm-browserify": "^1.1.2",
+    "vmsg": "^0.3.6",
+    "web-streams-polyfill": "^3.1.0",
     "wgxpath": "^1.2.0",
     "whatwg-fetch": "^2.0.3"
```

Comment:
Do you know if there's any way to check if any of these are unused? Might be nice to clear a few out if we can.

Reply:
Works for me! I started doing this a bit today. Doubt I'll finish, but it's nice to clear a few out.